src/camp_solver_data.F90

```diff
@@ -392,10 +417,10 @@ end subroutine solver_free
     procedure :: update_aero_rep_data
     !> Integrate over a given time step
     procedure :: solve
-    !> Reset the solver function timers
-    procedure, private :: reset_timers
-    !> Get the solver statistics from the last run
-    procedure, private :: get_solver_stats
+    procedure:: get_solver_stats
+    procedure:: export_solver_data_state
+    procedure:: join_solver_data_state
+    procedure:: export_solver_data_stats
```

Comment:
please add descriptions for what these functions do

Reply:
Added: 
```

    !> Create a file for saving output concentrations
    procedure:: init_export_solver_data_state
    !> Export output concentrations to calculate accuracy between CPU and GPU versions at checkGPU test
    procedure:: export_solver_data_state
    !> Join the files created by each MPI process at "export_solver_state" function into a single file.
    procedure:: join_solver_data_state
    !> Export execution time of GPU and CPU code to calculate speedups at TestMonarch.py
    procedure:: export_solver_data_stats
```